src/applications/gi/components/profile/Programs.jsx

```diff
@@ -57,6 +57,18 @@ export class Programs extends React.Component {
         text: 'Priority Enrollment',
         link: false,
       },
+
+      onlineOnly: {
```

Comment:
these properties are tied to the api response. e.g. onlineOnly maps to online_only